nachos/src/main/java/com/hootsuite/nachos/terminator/DefaultChipTerminatorHandler.java

```diff
@@ -53,12 +57,28 @@ public int findAndHandleChipTerminators(@NonNull ChipTokenizer tokenizer, @NonNu
                 switch (behavior) {
                     case BEHAVIOR_CHIPIFY_ALL:
                         selectionIndex = handleChipifyAll(textIterator, tokenizer);
+                        if (mOnChipAddListener != null) {
```

Comment:
@harhsoni1110 it seems that these 3 blocks do the same thing. Why not put it into a private method?